### PR TITLE
fix(EG-726): unable to load lab directly in new tab because labs store not persisted

### PR DESCRIPTION
**Pull request by willholt-tb**

### CODE REVIEW

File: packages/front-end/src/app/stores/labs.ts
Comment (simondib) [inline packages/front-end/src/app/stores/labs.ts:1]: This shouldn't be needed as the persist options are configured globally for all stores in the Nuxt config.

Comment (simondib) [reply packages/front-end/src/app/stores/labs.ts:1]: Ignore the above comment - I was mistaken, the Nuxt config is only setting the localStorage as the default storage mechanism. PR is good 👍

**Review by simondib — APPROVED** [review]

File: packages/front-end/src/app/stores/labs.ts
Comment (simondib) [reply packages/front-end/src/app/stores/labs.ts:1]: @willholt-tb The [plugin docs](https://prazdevs.github.io/pinia-plugin-persistedstate/frameworks/nuxt.html#usage) suggest this syntax: 

```
  {
    persist: true,
  }

```

I've noticed some other stores also need updating their syntax but I can do that separate to this PR.

Comment (willholt-tb) [reply packages/front-end/src/app/stores/labs.ts:1]: right, if the nuxt config sets it to localhost by default then in the store we can just `persist: true`, makes sense

**Review by simondib — APPROVED** [review]

File: packages/front-end/src/app/stores/labs.ts
Comment (simondib) [reply packages/front-end/src/app/stores/labs.ts:1]: Yep. Thanks for updating the other stores 👍

**Pull request merged**